app/utils/prepareAnswer.js

```diff
@@ -82,6 +93,24 @@ function prepareStatsDistrictAnswer(body, stateName) {
     return data
 }
 
+function prepareStateTestStat(body, stateName) {
+
+    let stateWise = body['states_tested_data'], stateDataIndex = -1
+    for (let index = 0; index < stateWise.length; index++) {
+        let currentData = stateWise[index]
+        if (currentData['state'].toLowerCase() === stateName.toLowerCase())
+            stateDataIndex = index
+    }
+    if (stateDataIndex === -1)
+        return '\n\u{26AA} Citizens Tested: Not Available'
+    else {
+        let stateData = stateWise[stateDataIndex]
+        let data = '\n\u{26AA} Citizens Tested: ' + numberWithIndianCommas(stateData['totaltested'])
```

Comment:
Please follow the same format as in the `all` command.
x citizens tested

Reply:
Okay i will change it 👍

Reply:
Changed